### PR TITLE
Install layer tests with CMake

### PR DESCRIPTION
**Pull request by alexander-shchepetov**
### Tickets:
 - **51810*
 - **60421*

### CODE REVIEW

File: tests/layer_tests/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+install(TARGETS ${TARGET_NAME} DESTINATION tests COMPONENT tests EXCLUDE_FROM_ALL)
```
Comment (ilya-lavrenov) [inline tests/layer_tests/CMakeLists.txt:1]: please, add copyright 
```
# Copyright (C) 2018-2021 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
#
```

Comment (ilya-lavrenov) [reply tests/layer_tests/CMakeLists.txt:1]: BTW, `TARGET_NAME` is not defined in this file

File: tests/layer_tests/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+project(layer_tests)
+
+install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/ DESTINATION layer_tests_openvino COMPONENT tests EXCLUDE_FROM_ALL)
```
Comment (ishariko) [inline tests/layer_tests/CMakeLists.txt:7]: please replace DESTINATION folder `layer_tests_openvino` to `tests\layer_tests_openvino`

**Review by ishariko — CHANGES_REQUESTED** [review]

**Review by ishariko — APPROVED** [review]

**Comment by azhogov** [comment]
CI passed: https://github.com/intel-innersource/frameworks.ai.openvino.ci.product-configs/pull/109

**Pull request merged**